drag_and_drop_v2/public/js/view.js

```diff
@@ -50,13 +51,21 @@
         if (item.is_placed) {
             style.left = item.x_percent + "%";
             style.top = item.y_percent + "%";
+            tabindex = -1;  // If an item has been placed it can no longer be interacted with,
```

Comment:
@cptvitamin Thanks - that answers my question perfectly.

@itsjeyd When an item is correctly placed, can you remove the tabindex completely instead of setting it to `-1`? Either approach should work for us here but as Mark mentions, that is preferable. If it's too much work, the current approach should also be fine and this is good to go :)